web/src/components/district/Councillor.js

```diff
@@ -30,10 +30,9 @@ class Councillor extends Component {
     // areas: PropTypes.array.isRequired
   }
 
-  homeUrl = 'https://cswbrian.github.io/district-councils-dashboard'
-
   render() {
     const { districts } = this.props
+    const homeUrl = process.env.REACT_APP_HOST_URI
```

Comment:
@wingkwong do I need to set up anything for the env variable before I deploy?

Reply:
just make sure REACT_APP_HOST_URI can be found in .env. I saw it has been included in .env-sample already.